src/AST-Core-Tests/RBMethodNodeTest.class.st

```diff
@@ -56,6 +56,20 @@ RBMethodNodeTest >> testCommentedMethodHasComments [
 
 ]
 
+{ #category : #'tests - primitives' }
+RBMethodNodeTest >> testIsPrimitive [
+	"see other testPrimitive... tests"
+	
+	| ast |
+	ast := (SmallInteger >> #+) ast.
+	self assert: ast isPrimitive.
+	ast := (Bitmap >> #primFill:) ast.
+	self assert: ast isPrimitive.
+	
+	ast := (Integer >> #+) ast.
+	self deny: ast isPrimitive.
+]
```

Comment:
This one looks the same than `testPrimitiveIsPrimitive`. Is that on purpose or a slip?